test/e2e/authorization.go

```diff
@@ -651,6 +665,41 @@ var _ = g.Describe("Authorization via admission-controller [RBAC] [Zalando]", fu
 				_, err := client.CoreV1().Pods(systemResource.Namespace).Create(context.Background(), systemResource, metav1.CreateOptions{DryRun: []string{"All"}})
 				gomega.Expect(err).To(gomega.MatchError(gomega.ContainSubstring("write operations are forbidden")))
 			})
+
+			// Not needed actually
+			// // TODO: need to create resource before deleting it
+			// g.It("should deny delete access in system namespace", func() {
+			// 	err := client.CoreV1().Pods(systemResource.Namespace).Delete(context.Background(), systemResource.Name, metav1.DeleteOptions{DryRun: []string{"All"}})
+			// 	gomega.Expect(err).To(gomega.MatchError(gomega.ContainSubstring("delete operations are forbidden")))
+			// })
+		})
+
+		// TODO: this is for manual/ememergency access (to be consistent let's rename it to "privleged" because this si now called "privielegd access" by the IAM team)
+		g.Context("as engineer user", func() {
+			var client *kubernetes.Clientset
+
+			g.BeforeEach(func() {
+				var err error
+
+				client, err = getEngineerClient(eksCluster, awsAccountID)
+				framework.ExpectNoError(err)
+			})
+
+			// these tests are similar to the ones for unprivileged (let's think about that) (also see remarks of the first test in the block below)
```

Comment:
Let's keep all access tested and not depend on tests for unprivileged even if this role is built upon it.